### PR TITLE
docs: Random generator from java Enums

### PR DESCRIPTION
**Pull request by kingthorin**
Update usage.md with an Option.option example.

Fixes datafaker-net/datafaker#888

### CODE REVIEW

**Comment by snuyanzin** [comment]
+1

File: docs/documentation/usage.md [hunk 1]
```diff
@@ -106,3 +106,22 @@ which can be handy for generating results multiple times.
...
+    MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY, SUNDAY
+}
+```
+
+To retrieve a random Day:
+
+```java
+private final Options opt = faker.options();
+opt.option(Day.class);
+```
```
Comment (kingthorin) [inline docs/documentation/usage.md:127]: Does this need the `=== "Java"` heading to render properly on the docs site?

Comment (snuyanzin) [reply docs/documentation/usage.md:127]: i guess it is required for specific tab naming, especially when there are several code snippets 

may be @bodiam has more insights

Comment (kingthorin) [reply docs/documentation/usage.md]: Okay I tweaked the text and example to now be within a `=== "Java"` block

Comment (bodiam) [reply docs/documentation/usage.md]: That is correct, but you should be able to run the docs locally right, but running `mkdocs serve` or something like that.

Comment (kingthorin) [reply docs/documentation/usage.md]: I have no idea, I've never tried. I have a Linux VM I could try on I guess.

Comment (bodiam) [reply docs/documentation/usage.md]: It's not that important, and if the formatting is a bit off, we'll fix it, no problem! Thanks for this PR!

Comment (kingthorin) [reply docs/documentation/usage.md]: Thanks! If you're good with the content, I'll go ahead and merge it?

Comment (bodiam) [reply docs/documentation/usage.md]: Yes :)

Comment (bodiam) [reply docs/documentation/usage.md]: I'll try to deploy this later, it's still something to automate.

**Comment by kingthorin** [comment]
Thanks

**Pull request merged**